PrebidMobile/PrebidMobileRendering/Utilities/UserAgentService.swift

```diff
@@ -22,11 +22,13 @@ public class UserAgentService: NSObject {
     
     public private(set) var userAgent: String = ""
     
-    private let webView = WKWebView()
+    private var webView: WKWebView?
```

Comment:
It should work well since the registry and `userAgent`  will be managed sequentially in the main thread. `userAgent` should be safe for reading from any thread. Alternatively, the value passed to `fetchUserAgent` completion handler should be used. 

@weibel also mentioned the "warming up" of the WKWebView object before runing the JS code in it. Is it possible that the result of fetching the ua will be different depending on the js execution time in the WebView lifecycle?

Reply:
I'm unsure about the warm up time. I need to run a few tests. The problems i have had might be becasue the webview was deallocated before the block could return as i did not think about the singleton issue that was solved with registering the views